osquery/logger/logger.cpp

```diff
@@ -443,6 +445,10 @@ Status logString(const std::string& message,
   return status;
 }
 
+namespace {
+const std::string kTotalQueryCounterMonitorPath("total.query.counter");
```

Comment:
AFAIK, it is much more convenient to have some hierarchy of the names separated by `'.'` in the key.
If your first part of the key is `total` that means you have (or could have potentially) the set of keys start with `total`. The same logic for the prefix `total.query`. The filesystem is a good analog for such kind of hierarchy.

And because of that would you like to change the names `total` and `query`? : 
```
query.total.count
```

Reply:
done. thanks